core-web/apps/dotcms-block-editor/project.json

```diff
@@ -49,12 +51,12 @@
                         {
                             "type": "initial",
                             "maximumWarning": "2mb",
-                            "maximumError": "5mb"
+                            "maximumError": "2mb"
                         },
                         {
                             "type": "anyComponentStyle",
-                            "maximumWarning": "6kb",
-                            "maximumError": "10kb"
+                            "maximumWarning": "200kb",
+                            "maximumError": "200kb"
```

Comment:
You are right, I think there is a better way to get only the styles for the components we use. Should I fix that here, or maybe create another issue for that? It can take time for some research.